web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -660,6 +660,12 @@ $.extend(fixmystreet.set_up, {
   },
 
   dropzone: function($context) {
+    if ('Dropzone' in window) {
```

Comment:
Ah, does this mean eb16dc0425f9f4bf89e0b5ad73bcbdad5a9a4b35 isn't needed any longer? Added in response to https://github.com/mysociety/fixmystreet/pull/2965#pullrequestreview-450683391 but if it's not loading the widget offline at all (fair enough), it presumably doesn't need to try and initialise it?

Reply:
That commit is still required as the bug was occurring during normal online usage not offline. This fixup commit (20a220d9b8ee177a9d9dc2df550e9c97ab5d1df7) disables the Dropzone widget when offline by reordering the `autoDiscover = true` code to be before `return` when `navigator.onLine == true`.